charts/hlf-k8s/values.yaml

```diff
@@ -24,14 +24,6 @@ fabric-ca-tools:
     tag: latest
     pullPolicy: IfNotPresent
 
-nginx-ingress:
-  enabled: false
-  controller:
-    config:
-        server-tokens: "false"
-    extraArgs:
-      enable-ssl-passthrough: ""
```

Comment:
I remember struggling a lot at first with that, maybe you should add a note on the README explaining that requirement from the ingress controller.

Reply:
Yep I'll add a note as this is important and someone could spend a lot of time on this issue.